hal/inc/interrupts_hal.h

```diff
@@ -83,6 +84,10 @@ typedef struct hal_interrupt_extra_configuration_t {
       uint8_t keepHandler  : 1;
     };
   };
+  struct {
+      bool appendHandler;
+      uint8_t priority;
```

Comment:
Let's give a more verbose name e.g. `chainPriority` or something like so that we are not mixing this up with IRQ priority etc.